packages/core/src/index.ts

```diff
@@ -1,19 +1,57 @@
 #!/usr/bin/env node
-import Fastify from "fastify";
+import Fastify, { FastifyInstance } from "fastify";
 import { TypeBoxTypeProvider } from "@fastify/type-provider-typebox";
 import { Static, Type } from "@sinclair/typebox";
 import { healthzRoute } from "./routes/healthz";
 import { catchAllRoute } from "./routes/catchAll";
 import { backdoorRoute } from "./routes/backdoor";
 import { Store } from "./store/store";
 import { MocksRegistry } from "./mocksRegistry/mocksRegistry";
+import avvio from "avvio";
+import { loadHandlers } from "./loader";
+import Registry from "./registry/registry";
+import Assert from "./registry/assert";
+import { start } from "@fastify/restartable";
+import { assertRoute } from "./routes/assert";
+import { as } from "./loader/rest";
 
-export const store = new Map<string, any>();
+export default as;
+
+type RegisterHandlerBodyType = {
+  name: string;
+  path: string;
+};
+
+type RegisterHandlerResponseType = {
+  status: number;
+  message: string;
+};
 
 const server = Fastify({
   logger: true,
 }).withTypeProvider<TypeBoxTypeProvider>();
 
+const app = avvio();
+
+const registry = new Registry();
+const assert = new Assert();
+
+app.use(loadHandlers, registry);
+
+export async function startApplication() {
+  const { listen } = await start({
+    protocol: "http", // or 'https'
+    hostname: "127.0.0.1",
+    port: 3100,
```

Comment:
We can have a ticket to add customisable variables `--port` `--protocol` etc

Reply:
Absolutely, I also want --ui that will spawn a web socket server for both dev tools and the admin page where we can change handlers at runtime.